ModuleNotFoundError: No module named 'stripe'

Запускаю `docker-compose up` и не получается запустить сервер.

```sh
club_app    |   File "/app/payments/service.py", line 1, in <module>
club_app    |     import stripe
club_app    | ModuleNotFoundError: No module named 'stripe'
```

После запуска `docker-compose up --build`:

```sh
club_app    |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
club_app    |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
club_app    |   File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
club_app    | ModuleNotFoundError: No module named 'debug_toolbar'
club_app    | make: *** [Makefile:22: docker-run-dev] Error 1
club_app exited with code 2
```

--- a/Makefile
+++ b/Makefile
@@ -38,6 +38,9 @@ lint:  ## Lint code with flake8
 requirements:  ## Generate requirements.txt for production
 	pipenv lock --requirements > requirements.txt
 
+dev_requirements:  ## Generate dev_requirements.txt for development
+	pipenv lock --dev-only --requirements > dev_requirements.txt
+
 migrate:  ## Migrate database to the latest version
 	pipenv run python3 manage.py migrate
 

--- a/dev.dockerfile
+++ b/dev.dockerfile
@@ -23,6 +23,10 @@ WORKDIR /app
 # install requirements into a separate layer
 COPY ./requirements.txt /app/requirements.txt
 RUN pip install -r requirements.txt
+
+COPY ./dev_requirements.txt /app/dev_requirements.txt
+RUN pip install -r dev_requirements.txt
+
 RUN python -c "import nltk; nltk.download('punkt')"
 
 # copy the code

--- a/dev_requirements.txt
+++ b/dev_requirements.txt
@@ -1,0 +1,20 @@
+-i https://pypi.org/simple
+asgiref==3.2.10; python_version >= '3.5'
+attrs==19.3.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+django-debug-toolbar==2.2
+django==3.0.4
+flake8==3.8.3
+iniconfig==1.0.0
+mccabe==0.6.1
+more-itertools==8.4.0; python_version >= '3.5'
+packaging==20.4; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+pluggy==0.13.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+py==1.9.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+pycodestyle==2.6.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+pyflakes==2.2.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+pyparsing==2.4.7; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
+pytest==6.0.1
+pytz==2020.1
+six==1.15.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+sqlparse==0.3.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+toml==0.10.1

